src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -1503,6 +1503,37 @@ public function testWitherWithStaticReturnType()
         $this->assertInstanceOf(Foo::class, $wither->foo);
     }
 
+    public function testCurrentFactoryInlining()
+    {
+        $container = new ContainerBuilder();
+        $container->register(Foo::class);
+
+        $container
+            ->register('inlined_current', Foo::class)
+            ->setFactory('current')
+            ->setPublic(true)
+            ->setArguments([[new Reference(Foo::class)]]);
+
+        $container
+            ->register('not_inlined_current', Foo::class)
+            ->setFactory('current')
+            ->setPublic(true)
+            ->setArguments([[new Reference(Foo::class), 123]]);
+
+        $container->compile();
+        $dumper = new PhpDumper($container);
+        $dump = $dumper->dump(['class' => 'Symfony_DI_PhpDumper_Service_CurrentFactoryInlining']);
+        file_put_contents(self::$fixturesPath.'/php/services_current_factory_inlining.php', $dump);
```

Comment:
@nicolas-grekas I think this was left here by mistake.

Reply:
Oops, fixed in 29054b1cc6